sdk/core/Azure.Core.TestFramework/src/MockResponse.cs

```diff
@@ -31,7 +31,7 @@ public MockResponse(int status, string reasonPhrase = null)
 
         public void SetContent(byte[] content)
         {
-            ContentStream = new MemoryStream(content);
+            ContentStream = new MemoryStream(content, 0, content.Length, true, true);
```

Comment:
Do we need it writable?

Reply:
We don't need it writable, I can change that.  We do need it to be publicly exposable.